Userland/Applications/SystemMonitor/main.cpp

```diff
@@ -242,6 +242,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     TRY(Core::System::unveil("/etc/passwd", "r"));
     TRY(Core::System::unveil("/res", "r"));
     TRY(Core::System::unveil("/proc", "r"));
+    TRY(Core::System::unveil("/sys", "r"));
```

Comment:
It should be enough to only unveil /sys/kernel/df.

Reply:
Well it's actually `/sys/kernel`, but I will fix that :)